src/admin-views/tribe-options-general.php

```diff
@@ -265,11 +265,40 @@
 ];
 
 if ( tribe_events_views_v2_is_enabled() ) {
+	// Push the control to the Filters tab
 	add_filter( 'tribe-event-filters-settings-fields', function ( $fields ) use ( $filter_activation ) {
 		$fields += $filter_activation;
 		return $fields;
 	} );
 } else {
+	/**
+	 * Filters the text for the "automatic" option.
+	 *
+	 * @since TBD
+	 *
+	 * @param string the displayed text.
+	 */
+	$automatic_text = apply_filters(
+		'tribe_events_liveupdate_automatic_label_text',
+		__( 'Enabled: datepicker selections automatically update calendar views.', 'the-events-calendar')
+	);
+	/**
+	 * Filters the text for the "manual" option.
+	 *
+	 * @since TBD
+	 *
+	 * @param string the displayed text.
+	 */
+	$manual_text    = apply_filters(
+		'tribe_events_liveupdate_manual_label_text',
+		__( 'Disabled: users must click Find Events to search by date.', 'the-events-calendar')
+	);
+
+	$filter_activation['liveFiltersUpdate']['options']['automatic'] = $automatic_text;
+	$filter_activation['liveFiltersUpdate']['options']['manual'] = $manual_text;
+	$filter_activation['liveFiltersUpdate']['label'] = esc_html__( 'Live Refresh', 'the-events-calendar' );
+
+	// Insert the control
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 676f03860de41800f0cbf5c1f69a5f4a</sub>